### PR TITLE
chore: Use official code group support to replace custom tab component

### LINKED ISSUE #221: bug: Tab is not displayed correctly in dark mode
image

### LINKED ISSUE #224: fix(logo): Replace tensorchord logo with envd
Screenshot 2022-12-22 at 09 39 01

### PR DESCRIPTION
**Pull request by VoVAllen**
Fix #221

Also fix #224  #220

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -42,3 +42,4 @@ __pycache__
 
 # Intermediate result, should not update to git 
 hack/api/
+docs/.vitepress/cache
```
Comment (gaocegege) [inline .gitignore:45]: Please add docs-zh too

**Comment by gaocegege** [comment]
Please ping me if the conflicts are resolved.

**Comment by VoVAllen** [comment]
@gaocegege resolved

**Review by kemingy — APPROVED** [review]
/lgtm

**Pull request merged**